include/api/CDataFrameTrainBoostedTreeRunner.h

```diff
@@ -115,6 +116,7 @@ class API_EXPORT CDataFrameTrainBoostedTreeRunner : public CDataFrameAnalysisRun
 
     std::string m_DependentVariableFieldName;
     std::string m_PredictionFieldName;
+    double m_trainingPercent;
```

Comment:
nit: consistency and naming conventions
```suggestion
    double m_TrainingPercent;
```